resources/configure.rb

```diff
@@ -0,0 +1,245 @@
+unified_mode true
+# this is what helps the various resources find each other
+property :instance_name, String
+
+# if you override one of these, you should probably override them all
+property :path_home,    String, default: '/usr/share/elasticsearch'
+property :path_conf,    String, default: '/etc/elasticsearch'
+property :path_data,    String, default: '/var/lib/elasticsearch'
+property :path_logs,    String, default: '/var/log/elasticsearch'
+property :path_pid,     String, default: '/var/run/elasticsearch'
+property :path_plugins, String, default: '/usr/share/elasticsearch/plugins'
+property :path_bin,     String, default: '/usr/share/elasticsearch/bin'
+
+property :template_elasticsearch_env, String, default: 'elasticsearch.in.sh.erb'
+property :cookbook_elasticsearch_env, String, default: 'elasticsearch'
+
+property :template_jvm_options, String, default: 'jvm_options.erb'
+property :cookbook_jvm_options, String, default: 'elasticsearch'
+
+property :template_elasticsearch_yml, String, default: 'elasticsearch.yml.erb'
+property :cookbook_elasticsearch_yml, String, default: 'elasticsearch'
+
+property :template_log4j2_properties, String, default: 'log4j2.properties.erb'
+property :cookbook_log4j2_properties, String, default: 'elasticsearch'
+
+property :logging, Hash, default: {}.freeze
+property :java_home, String
+
+# other settings in /etc/default or /etc/sysconfig
+property :memlock_limit, String, default: 'unlimited'
+property :max_map_count, String, default: '262144'
```

Comment:
Any reason to make these strings?

Reply:
They were Strings before. That's all. Happy to have both types here.